src/components/ebay-drawer-dialog/component.js

```diff
@@ -20,15 +20,20 @@ export default {
 
     handleTouchStart(event) {
```

Comment:
Might be worth briefly documenting at some point what is actually happening here with these touch events, at a high level.

Reply:
Yes. Overall its basically detecting if the area was tapped and dragged up/down by a certain amount. 
the change here was that native events don’t support forEach. I could have used them in a prototype but i kinda like just using the base for loop

Reply:
> its basically detecting if the area was tapped and dragged up/down by a certain amount.

Cool. Please create a task or note for that.

Reply:
Will do